packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts

```diff
@@ -157,10 +153,6 @@ const init = async () => {
   Container.set(CurrencyHelper, currencyHelper);
```

Comment:
Are the CurrencyHelper and StripeHelper still needed if not checking subscriptions?

Reply:
+1 to above questions. Also `authFirestore` references, do we use that for anything else?

Reply:
Thank you for catching that.  I will check.  The thing with `Container` is that one cannot tell if something else in the dependency tree will reach in to grab something.